web/client/cypress/e2e/validator.cy.js

```diff
@@ -0,0 +1,93 @@
+/// <reference types="cypress" />
+
+context('GTFS Validator - Core Workflow', () => {
+  let numReportCalls = 0;
+
+  beforeEach(() => {
+    cy.visit('https://gtfs-validator.mobilitydata.org/')
+  });
+
+  it('Validate GTFS .zip file', () => {
+    const url = 'https://developers.google.com/static/transit/gtfs/examples/sample-feed.zip';
+    const jobId = '8f6be6fb-1fee-41f8-b401-b2b4b552e177';
+
+    // Setup intercept aliases
+    cy.intercept(
+        'POST',
+        'https://gtfs-validator-web-mbzoxaljzq-ue.a.run.app/create-job',
+        (req) => {
+          req.headers['x-custom-headers'] = 'added by cy.intercept';
+          req.reply({
+            statusCode: 200,
+            statusMessage: "OK",
+            headers: {
+              "x-custom-header": 'mgunn',
+            },
+            body: {
+              jobId: jobId
+            }
+          });
+        }
+      )
+      .as('createJob');
+
+
+    cy.intercept(
+      'HEAD',
+      'https://gtfs-validator-results.mobilitydata.org/*/report.html',
```

Comment:
Let's use `${jobId}` here instead of `*` so we're asserting that the `jobId` being returned by `/create-job` is the one the client is going after

Reply:
i was running into an issue here bc the first 404 HEAD response doesn't include the `jobId`, it includes `_waiting_`. But i've updated the test to assert that the `jobId` is found where expected.